Assets/Scripts/CopilotMechanics/SpaceGirlPassive.cs

```diff
@@ -0,0 +1,50 @@
+﻿using System.Collections;
+using System.Collections.Generic;
+using UnityEngine;
+
+public class SpaceGirlPassive : CopilotPassiveMechanic
+{
+
+    public float healthPercent = 25f;//Percent Health threshold
+    public int damageIncrease;//Amount of damage added to base damage
+
+    private bool damageNormalized;
+
+    private void Start()
+    {
+        damageNormalized = true;
+    }
+
+    private void Update()
```

Comment:
Remove the Update Method and instead create a Method called CheckPassiveActivation,  in here you will check if the Player health is currently lower than the threshold, and if so you will call the AddEffect method in PlayerInfo where you will pass in a SatusEffect of type Focus, the damage increase variable, and the Timer Effect variable. You will need to subscribe this method to the PlayerInfo damageEvent. Check the MushRoomFriendPassive script  on the Start method line 16 to learn how to do this.